rest/taskrouter/activities/instance/delete/example-1/example-1.5.x.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library
+
+use Twilio\Rest\Client;
```

Comment:
I saw you changed this in all the snippets. Any reason why you decided not to use the global namespace for every snippet? This works only because we are not using namespaces in this context, might fail if the snippet is copied into a file that is actually using a namespace